diff --git a/src/tools/qtgrpcgen/clientdeclarationprinter.cpp b/src/tools/qtgrpcgen/clientdeclarationprinter.cpp
index 6ae83b2..167e07a 100644 (file)
--- a/src/tools/qtgrpcgen/clientdeclarationprinter.cpp
+++ b/src/tools/qtgrpcgen/clientdeclarationprinter.cpp
@@ -8,7 +8,6 @@
 #include <google/protobuf/io/printer.h>
 #include <google/protobuf/io/zero_copy_stream.h>
 #include <google/protobuf/stubs/common.h>
-#include <google/protobuf/stubs/logging.h>

 #include <string>
 #include <unordered_set>
